base_layer/wallet_ffi/src/lib.rs

```diff
@@ -4619,6 +4619,65 @@ pub unsafe extern "C" fn comms_list_connected_public_keys(
     }
 }
 
+/// Gets the length of the public keys vector
+///
+/// ## Arguments
+/// `*const TariPublicKeys` - Pointer to emoji set
```

Comment:
public keys?

Reply:
That reminds me that there are more typos like this. But going over ~176 functions is not part of this PR.